test/todo_cli_test.exs

```diff
@@ -27,22 +28,68 @@ defmodule TodoCliTest do
   test "Update list name" do
     list = TodoCli.Repo.insert!(%List{title: "My poorly named list"})
     bad_list = TodoCli.Repo.get!(List, list.id)
-    IO.inspect bad_list
+
     from(l in List, where: l.id == ^list.id, update: [set: [title: "Better title"]])
     |> TodoCli.Repo.update_all([])
+
     new_list = TodoCli.Repo.get!(List, list.id)
     assert new_list.id == list.id
   end
 
   test "delete list by name" do
     list = TodoCli.Repo.insert!(%List{title: "House chores"})
-    result = case TodoCli.Repo.delete(list) do
-      {:ok, struct}  ->
-      "Deleted successfully"
 
-      {:error, _reason} ->
-      "Error"
-    end
+    result =
+      case TodoCli.Repo.delete(list) do
+        {:ok, struct} ->
+          "Deleted successfully"
+
+        {:error, _reason} ->
+          "Error"
+      end
+
     assert result == "Deleted successfully"
   end
+
+  test "create list and associated task" do
+    list = List.changeset(%List{}, %{title: "My new list"})
+
+    list =
+      if list.valid? do
+        TodoCli.Repo.insert!(list)
+      else
+        list.errors
+      end
```

Comment:
If a user doesn't provide a title, the changeset will be invalid. Could the user provide an empty string as a title, yes. But these tests are not testing the functions I created. They are testing how the repo works. I did this because I was not sure how the queries worked in the beginning of this project. The last set of tests where the test name is, "my_function/1" test the functions I wrote after gaining a better understanding of Ecto. These preliminary tests can probably be deleted since they do not test any actual implementation of code I wrote.